### DISCUSSION TITLE
mdi container and keydown events in maui desktop app

### DISCUSSION THREAD

**Discussion by hintsofttech** [dotnet/maui#1449] [Ideas]
please talk when maui desktop app will support mdi container and keydown events like winform does ?

**Comment by mouralabank** [comment]
Can we have multiple document interface support (MDI) or something similar in MAUI? For Desktop applications is something very important and brings several benefits, flexibility and facilities.

Generally desktop platforms have many tools and every user has a way to assemble their workspace. With MDI the user can open multiple tools, move, minimize, maximize, resize and configure their work area as desired and is more useful in each situation.

**Reply by mouralabank** [reply]
Other benefits of MDI for desktop applications:

Improved organization: MDI allows users to organize multiple documents or windows within a single parent window, making it easier to manage and navigate between them.
More efficient use of screen space: By using MDI, desktop applications can maximize screen real estate by consolidating multiple windows into a single parent window.
Better task management: Users can switch between open documents or windows more easily, as they are all contained within a single parent window.
Reduced clutter: MDI can help reduce desktop clutter by consolidating multiple windows into a single parent window, making it easier to find the document or window you need.
Consistent user experience: By using a common interface for multiple documents or windows, MDI can provide a consistent user experience across different parts of the application.
MDI can help improve productivity and user experience in desktop applications by providing a more efficient and organized way to manage multiple documents or windows
